app/assets/stylesheets/application.scss

```diff
@@ -7,19 +7,22 @@
 @import "typography";
 
 // Mixins
+@import "mixins/css3";
```

Comment:
You can use @import "css3";  here - no need to add this file to the repo, as it is [part of the govuk frontend toolkit](https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_css3.scss) (in the same way as typography above).